src/EppoClient.php

```diff
@@ -115,16 +94,16 @@ public static function init(
                 $baseUrl
             );
 
-            $configRequester = new ExperimentConfigurationRequester($apiWrapper, $configStore);
+            $configLoader = new FlagConfigurationLoader($apiWrapper, $configStore);
             $poller = new Poller(
                 self::POLL_INTERVAL_MILLIS,
                 self::JITTER_MILLIS,
-                function () use ($configRequester) {
-                    $configRequester->fetchAndStoreConfigurations();
+                function () use ($configLoader) {
+                    $configLoader->fetchAndStoreConfigurations();
```

Comment:
The poller uses sleep to wait between polls so we need to run it in a separate process. It's been awhile since I've worked with PHP but it's still generally single-threaded and the `EppoClient init` and `getAssignment` calls will be made during a short-running script (executed once for each request). I'm not well-versed enough in modern php server development to know the best/most canonical way to implement polling - is there an Eppo customer using PHP we could reach out to?

Reply:
Oh right, I forgot PHP is weird like that. Does this mean calls to get assignments will kick off a fetch _every_ time?

Vimeo is (was?) heavy with PHP, although they are using an older version of both PHP and our SDK.